Gopkg.toml

```diff
@@ -12,8 +12,10 @@
   name = "github.com/btcsuite/websocket"
 
 [[constraint]]
-  branch = "master"
+  branch = "SplitTx"
```

Comment:
If you plan to merge into master, should the branch generated for the dependencies still be constrained to `SplitTx`?

Reply:
Yes this is temporary and will be changed prior to merge

Reply:
it's temporary. as this PR depends on another PR to pass travis ci tests